reference/display.md

```diff
@@ -247,9 +248,8 @@ closed. Here is an example :
   wb_display_draw_polygon(display,px,py,4); // draw a diamond
 ```
 
-`wb_display_draw_text()` draws an ASCII text from the (`x`,`y`) coordinate. The
-font used to display the characters has a size of 8x8 pixels. There is no extra
-space between characters.
+`wb_display_draw_text()` draws an ASCII text from the (`x`,`y`) coordinate.
+The used font is defined by the [`wb_display_set_font()`](#wb_display_set_color) function.
```

Comment:
Shouldn't we say "the font used for drawing the text" rather than "the used font"?